src/utils/transaction-view.js

```diff
@@ -109,9 +109,14 @@ module.exports = {
       }
 
       const transactionType = element.transaction_type && element.transaction_type.toLowerCase()
-      if (transactionType === 'refund' || transactionType === 'dispute') {
+      if (transactionType === 'dispute') {
+        console.log('\n\n\nDispute transaction type found. Element state=', element.state)
+        console.log('\n\n\n')
+      }
```

Comment:
Can we need these logs?

Reply:
No, we don't. I'll remove them before publishing the PR. 👍

Reply:
Logs removed, now. Thank you